admin/RunIncrementalJSONDump

```diff
@@ -23,13 +23,25 @@ source admin/config.sh
 . ./admin/functions.sh
 make_temp_dir
 
+# Create a clean test database from which to dump foreign keys.
+export REPLICATION_TYPE=3 # RT_STANDALONE
+./script/create_test_db.sh TEST
+
+FK_DUMP="$TEMP_DIR"/foreign_keys
+./script/dump_foreign_keys.pl \
+    --database TEST \
+    --output "$FK_DUMP"
+
 echo Making incremental JSON dumps
 ./admin/DumpIncrementalJSON \
     --output-dir "$TEMP_DIR" \
     --compress \
     --database READWRITE \
+    --foreign-keys-dump "$FK_DUMP" \
     || exit $?
 
+rm -f "$FK_DUMP"
```

Comment:
Is is at all potentially problematic that on error this will exit above (I assume) without removing the dump file?

Reply:
Added `EXIT` traps.